### DISCUSSION TITLE
Custom thumbnail (og:image/twitter:image) image for every page

### DISCUSSION THREAD

**Discussion by aapwikip** [requarks/wiki#6887] [Help / Questions]
While we have set default og:image/twitter:image for preview or thumbnail image for all articles; using html::head injection

We now want to try setting a Custom preview image for every article  i.e. overwriting the default og:image

a. We tried mention the same meta og:image html tag in the markdown page

```
<html>
  <head>
<meta property="og:image" content="https://aamaadmiparty.wiki/delhianganwadikitchen.jpeg" />
<meta name=”twitter:image” content=”https://aamaadmiparty.wiki/delhianganwadikitchen.jpeg” />
    </head>
</html>
```

b. We tried overwriting with javascript dynamically from script tab

```
document.querySelectorAll('meta[property=og\\:image]')[0].setAttribute('content', 'https://xxx.wiki/abc.jpeg')
document.querySelectorAll('meta[property=twitter\\:image]')[0].setAttribute('content', 'https://xxx.wiki/abc.jpeg')
```

But NONE worked. Please suggest how to achieve it @NGPixel

**Comment by LagClear** [comment]
Is this possible yet?

**Reply by aapwikip** [reply]
Not yet

**Comment by SinTan1729** [comment]
This will not work for everyone, but if you use Caddy to serve your WikiJS instance, I wrote a plugin to do just this.
https://github.com/SinTan1729/caddy-wikijs-metatags
